packages/cactus-plugin-ledger-connector-sawtooth-socketio/package.json

```diff
@@ -2,6 +2,9 @@
   "name": "@hyperledger/cactus-plugin-ledger-connector-sawtooth-socketio",
   "version": "1.1.2",
   "license": "Apache-2.0",
+  "main": "dist/index.js",
```

Comment:
@outSH I recommend keeping the conventions of the other packages when it comes to declaring the paths & configuring the compiler (tsconfig.json file). That will avoid a big chunk of the issues like when some packages do not get published due to strange bugs: https://github.com/hyperledger/cactus/issues/2069 or other kinds of automation scripts failing if they make assumptions that some of these paths are uniform everywhere. If there is a specific reason for not going uniform with the package configuration properties then that's a different question of course but if it's all the same to you then I'd highly recommend changing it to match the rest of the packages.

The same goes for the tsconfig.json file, the include paths there ideally would match the rest of the tsconfig.json files which would lead to the compiler output being structured the same way as well.

Reply:
BTW the same applies to all `*-socketio-*` connectors, so maybe it is better to handle in separate PR

Reply:
> @petermetz Sorry, I don't understand, can you clarify the conventions you want to keep (would be even nice to have them documented somewhere)? Are the paths supposed to be like `dist/lib/main/typescript/index.js`? If so, it doesn't seem much work and I could do it in this PR I suppose.

@outSH Yeah, that's one of them, but in general every single property in the package.json file that does not have a specific reason (like contributor list or package name/description) to be different should be the same, uniform configuration (same for tsconfig.json files).

Agreed 100% that this should be documented. Best I could do earlier is to make the changes myself last year but then all of that got deleted by others from my own PR so in the end we got what we got.
I'll create a task in the tracker to have a custom check that enforces a set of rules for these files. That way it's not just a markdown file that will almost nobody ever read but instead people will get a failing CI check if anything is wrong with their packages. This will also make our review process more scalable as we continue to add more packages in the future.

---

> BTW the same applies to all *-socketio-* connectors, so maybe it is better to handle in separate PR

 Yeah I'm slowly but surely trying to get us to a state where all packages are as uniform as possible in terms of build configuration to make everybody's lives easier and this would be a part of that so thank you very much for offering. I'm totally fine with doing a separate and bigger PR with all of them, so I'll approve this one as-is and leave it up to you how you want to do it for this specific package. Thank you again!